### PR TITLE
[bazel] Bazel support

### PR DESCRIPTION
**Pull request by dereklowlind**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
New feature request for Bazel
https://github.com/withfig/autocomplete/issues/255

**Additional info:**
this copy works but has an issue where it will only pickup Bazel target suggestions from one Bazel BUILD file. Not sure if using the custom flag in the generator over postProcess would make a difference or not. I think the script command is returning before the for loop opens and prints the contents from all of the Bazel build files to the command line.

### CODE REVIEW

File: dev/bazel.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      }
+    }
+    return targets;
+  },
+};
+
+export const completionSpec: Fig.Spec = {
+  name: "bazel",
+  description: "Bazel the build system!",
+  options: [
```
Comment (QuiiBz) [inline dev/bazel.ts:40]: This should be `subcommands`, not `options`

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this PR! I'm not familiar at all with bazel, but does those commands are the most important ones? I see a lot more commands and options in the [CLI reference](https://docs.bazel.build/versions/main/command-line-reference.html), so it would be amazing to have them all! (or at least the most important ones)

File: dev/bazel.ts (hunk 1 above)
Comment (dereklowlind) [reply dev/bazel.ts]: Would it be better to split the bazel support into two PRs? In issue https://github.com/withfig/autocomplete/issues/255 rolandtshen asks for run, build, and test with a generator for recommending targets. I was thinking we get this "mvp" to rolandtshen to test out and then add in the important/rest of the commands in another PR. I'm not a bazel power user so I might need to find one to help out (maybe from their github) on the what commands are important and testing the tool.

Comment (Monkeyank) [reply dev/bazel.ts]: I've built out all the other options/commands, so I can add these in if you'd like!

Comment (QuiiBz) [reply dev/bazel.ts]: @Monkeyank that would definitely help a lot! Could you push a commit directly here, so we won't need to have two PRs for bazel?

**Comment by QuiiBz** [comment]
@Monkeyank could you commit the other subcommands/options/args that you made, so we can get this merged in?

**Comment by Monkeyank** [comment]
Don't seem to have perms to commit directly to the PR. Also I have only been able to add shared options so far

**Comment by QuiiBz** [comment]
> Don't seem to have perms to commit directly to the PR. Also I have only been able to add shared options so far

Alright, then you can open another PR later when done. We can merge this one to already get the skeleton, and then merge your PR to complete `bazel`!

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**